packages/fast-components-react-base/src/checkbox/checkbox.spec.tsx

```diff
@@ -29,6 +29,12 @@ describe("checkbox", (): void => {
 
     const inputSelector: string = `.${managedClasses.checkbox_input}`;
 
+    test("should have correct element type checkbox", () => {
```

Comment:
Should we test for the root as well here, which is div?

Reply:
Suggest within a different test block if we do.

Reply:
To a point that makes sense. However, certain other wrappers around this would break accessibility. `div` => `span` is fine, but what if someone changed it to something else with a semantic role? I would say that *does* impact accessibility. Perhaps we don't want to or feel the need to check for it, but I do think that changing the root element to anything but a non-semantic element would break accessibility.

Reply:
If we do a test for this, let's do it separately because the goal is different. Closing this out